src/FSharpLint.Core/Application/Configuration.fs

```diff
@@ -668,7 +672,8 @@ let loadConfig (configPath:string) =
 let defaultConfiguration =
     let assembly = typeof<Rules.Rule>.GetTypeInfo().Assembly
     let resourceName = Assembly.GetExecutingAssembly().GetManifestResourceNames()
-                       |> Seq.find (fun resourceFile -> resourceFile.EndsWith("fsharplint.json", System.StringComparison.Ordinal))
+                       |> Seq.tryFind (fun resourceFile -> resourceFile.EndsWith("fsharplint.json", System.StringComparison.Ordinal))
+                       |> Option.defaultWith (fun () -> failwith "Could not get resource name")
```

Comment:
@webwarrior-ws now that we're changing these lines, let's indent them properly (4 spaces, not more)

Reply:
Done.